ci/env/install-dependencies.sh

```diff
@@ -316,11 +312,18 @@ install_pip_packages() {
     if [ "$status" != "0" ]; then
       echo "${status}" && return 1
     fi
-  fi
 
-  # Default requirements
```

Comment:
we dont need this MINIMAL_INSTALL anymore?

Reply:
requirements.txt includes `ray[all]` which was `requirements_default` before, so this should be fine

Reply:
Wait but we don't want to install `ray[all]` if MINIMAL_INSTALL is true right?

Reply:
I think the check in line 300 makes (and made) this check redundant

Reply:
Ah got it! thanks for pointing that out...that line was collapsed in the PR review